services/docker-compose.devel.yml

```diff
@@ -63,15 +63,15 @@ services:
 
   static-webserver:
     volumes:
-      - ./web/client/source-output:/static-content
+      - ./static-webserver/client/source-output:/static-content
     environment:
       - SERVER_LOG_LEVEL=info
 
   webserver:
     volumes:
       &webserver-volumes-dev
       - ./web/server:/devel/services/web/server
-      - ./web/client/source-output:/devel/services/web/client
+      - ./static-webserver/client/source-output:/devel/services/static-webserver/client
```

Comment:
Do we really need access to the client folder anymore?

Reply:
nope, removed.